namespaces/live-1.cloud-platform.service.justice.gov.uk/emile-sample-app-dev/resources/ecr.tf

```diff
@@ -3,7 +3,7 @@ terraform {
 }
 
```

Comment:
I'd recommend putting this provider block in your main.tf file. That will avoid conflicts if you ever add more resources to this namespace.